app/bundles/CoreBundle/DependencyInjection/EnvProcessor/MauticConstProcessor.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Mautic\CoreBundle\DependencyInjection\EnvProcessor;
+
+use Symfony\Component\DependencyInjection\EnvVarProcessorInterface;
+
+class MauticConstProcessor implements EnvVarProcessorInterface
```

Comment:
Needed a custom env var processor because Symfony's built-in `const` processor didn't seem to work as expected https://symfony.com/doc/current/configuration/env_var_processors.html#built-in-environment-variable-processors